tools/data/activitynet/generate_rawframes_filelist.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
Where is this file used?

Reply:
to generate anet_train_video.txt, ... which are used for anet finetuning